src/ralph/accounts/views.py

```diff
@@ -1,7 +1,28 @@
 # -*- coding: utf-8 -*-
+from django.views.generic import TemplateView
+from django.conf import settings
+from django.contrib.auth import REDIRECT_FIELD_NAME
+from django.contrib.auth.views import redirect_to_login
+
+from ralph.accounts.admin import UserInfoMixin
 from ralph.admin.mixins import RalphTemplateView
 
 
 class UserProfileView(RalphTemplateView):
-
     template_name = 'ralphuser/user_profile.html'
+
+
+class CurrentUserInfoView(UserInfoMixin, TemplateView):
+    template_name = 'ralphuser/my_equipment.html'
+
+    def dispatch(self, request, *args, **kwargs):
+        if not request.user.is_authenticated():
```

Comment:
wouldn't it be better to use https://docs.djangoproject.com/en/1.8/topics/auth/default/#the-login-required-decorator ?